pkgs/data/fonts/tipa/default.nix

```diff
@@ -1,10 +1,11 @@
 { stdenv, fetchurl }:
 
-stdenv.mkDerivation {
-  name = "tipa-1.3";
+stdenv.mkDerivation rec {
+  pname = "tipa";
+  version = "1.3";
 
   src = fetchurl {
-    url = "mirror://debian/pool/main/t/tipa/tipa_1.3.orig.tar.gz";
+    url = "mirror://debian/pool/main/t/tipa/${pname}_${version}.orig.tar.gz";
```

Comment:
Please try to substitute as little as possible from the URL to keep complexity of download URLs low.